_layouts/search.html

```diff
@@ -0,0 +1,6 @@
+---
+layout: default
+---
+<div class="search">
+	{{ content }}
```

Comment:
Same thing mentioned above. Styling seems to be 2 space indentation and use 4 spaces only when wrapping long lines of code.